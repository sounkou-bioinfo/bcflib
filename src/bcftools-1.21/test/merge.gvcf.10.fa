>chr1
ACGT
>chr2
ACGTACGT
