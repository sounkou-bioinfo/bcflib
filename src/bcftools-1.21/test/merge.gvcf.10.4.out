##fileformat=VCFv4.2
##FILTER=<ID=PASS,Description="All filters passed">
##contig=<ID=chr1,length=248956422>
##contig=<ID=chr2,length=248956422>
##INFO=<ID=END,Number=1,Type=Integer,Description="Stop position of the interval">
##FORMAT=<ID=GT,Number=1,Type=String,Description="Genotype">
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	a	b
chr1	1	.	A	<*>,C	.	.	.	GT	0/0	0/2
chr1	2	.	C	<*>	.	.	.	GT	0/0	./.
chr2	1	.	A	<*>	.	.	END=2	GT	0/0	./.
chr2	3	.	G	C,<*>	.	.	.	GT	0/1	./.
chr2	3	.	G	A,<*>	.	.	.	GT	./.	0/1
chr2	4	.	T	<*>	.	.	END=6	GT	0/0	0/0
chr2	7	.	G	<*>	.	.	END=8	GT	./.	0/0
